src/ice_setup_step.F90

```diff
@@ -132,7 +132,7 @@ subroutine ice_array_setup(mesh)
  allocate(stress_iceoce_x(n_size), stress_iceoce_y(n_size))    
  allocate(U_w(n_size), V_w(n_size))   ! =uf and vf of ocean at surface nodes
 #if defined (__oasis)
-  allocate(oce_heat_flux(n_size), ice_heat_flux(n_size))
+  allocate(oce_heat_flux(n_size), ice_heat_flux(n_size), enthalpyoffuse(n_size))
```

Comment:
Does it make sense to only to wrap it in the `if defined (__oifs)` statement to avoid creation of an empty variable when not in use? Similar to how it's done in https://github.com/FESOM/fesom2/pull/70/files